source/tutorials/phylogeny.rst

```diff
@@ -119,14 +119,15 @@ plugin.
 
 *Reference based alignments*
 
-There are a variety of tools such as `PyNAST`_) (using `NAST`_), `Infernal`_,
-and `SINA`_, etc., that attempt to reduce the amount of ambiguously aligned
-regions by using curated reference alignments (e.g. `SILVA`_. Reference
+There are several tools that attempt to reduce the amount of ambiguously aligned
+regions by using curated reference alignments. Traditional, de novo alignment methods mututally align a set of unaligned sequences to create a multiple sequence alignment (MSA) from scratch. Re-running these methods with additional sequences will create MSAs with varying numbers of columns and assignments of bases to each column. These alignments is therefore incompatible with one another and may not be joined through concatenation.
+
+Reference based alignments, on the other hand, are meant to add sequences to an existing alignment. Alignments computed using reference based alignment tools always have widths identical to the reference alignment and maintain the meaning of each column. Therefore, these alignments may be concatenated.
```

Comment:
Is this always true? What if a query sequence has a unique and large insertion relative to the reference? I would expect the existing alignment to change to compensate. The individual coordinates could still be relative to the reference, but the number of columns would be larger.

Reply:
I ported this text over verbatim from the community tutorial written by @epruesse so presumably this is the case at least for SINA, but I suppose this behavior might be tool-specific. 

We could also just drop this info altogether if it does not hold for other methods. @epruesse any comment on this?